logzero/__init__.py

```diff
@@ -431,6 +426,37 @@ def logfile(filename, formatter=None, mode='a', maxBytes=0, backupCount=0, encod
         logger.addHandler(rotating_filehandler)
 
 
+def remove_internal_loggers(logger_to_update, disableStderrLogger=True):
```

Comment:
perhaps add a default value like `logger_to_update=logger`?

Reply:
How about we just make it private? I don't think this method should ever be used outside of `logzero` internals. I think the real, proper design would be to do what I described in Discord and make logzero loggers a class where this method would be a private class method, instead.